dll/win32/shell32/wine/shell32_main.c

```diff
@@ -1159,6 +1200,18 @@ static INT_PTR CALLBACK AboutDlgProc( HWND hWnd, UINT msg, WPARAM wParam, LPARAM
 
                 // Preload the ROS bitmap
                 hLogoBmp = (HBITMAP)LoadImage(shell32_hInstance, MAKEINTRESOURCE(IDB_REACTOS), IMAGE_BITMAP, 0, 0, LR_DEFAULTCOLOR);
+#ifdef __REACTOS__
+                if(IsVersionWorkstation() == TRUE)
+                {
+                   // Load Workstation Bitmap
+                   hLogoBmp = (HBITMAP)LoadImage(shell32_hInstance, MAKEINTRESOURCE(IDB_REACTOS_WORKSTATION), IMAGE_BITMAP, 0, 0, LR_DEFAULTCOLOR);
+                }
+                else
+                {
+                   // Load Server Bitmap
+                   hLogoBmp = (HBITMAP)LoadImage(shell32_hInstance, MAKEINTRESOURCE(IDB_REACTOS_SERVER), IMAGE_BITMAP, 0, 0, LR_DEFAULTCOLOR);
+                }
+#endif
```

Comment:
At the code above the ReactOS logo bitmap is loaded by the `LoadImage` call yet after that you're discarding and overwriting it with a new one based on Workstation/Server setup condition. This is a waste of time, why don't you just load the logo bitmap based on the condition alone? Also why is this chunk of code wrapped within `__REACTOS__` (same as the other one)?

Reply:
"why is this chunk of code wrapped within __REACTOS__"

Because this is a file from Wine, which you aren't supposed to modify. I was told that when adding new code to it, it should be wrapped with it. You can see more of these throughout the code.

"you are discarding and overwriting the code above with a new one, why don't you just load the logo bitmap based on the condition alone?"

To preserve the original Wine code. I wasn't sure what to do with it.

Reply:
I'll leave it commented for now.

Reply:
> Because this is a file from Wine, which you aren't supposed to modify

See my previous comment, our shell32 is forked, and not being synched with Wine.